spec/unit/middleware/rate_limiter_v2_api_spec.rb

```diff
@@ -0,0 +1,373 @@
+require 'spec_helper'
+
+module CloudFoundry
+  module Middleware
+    RSpec.describe RateLimiterV2API do
+      let(:middleware) do
+        RateLimiterV2API.new(
+          app,
+          {
+            logger:                            logger,
+            per_process_general_limit:         per_process_general_limit,
+            global_general_limit:              global_general_limit,
+            per_process_admin_limit: per_process_admin_limit,
+            global_admin_limit:      global_admin_limit,
+            interval:                          interval,
+          }
+        )
+      end
+      let(:REQUEST_COUNTER_V2_API) { double }
```

Comment:
Same as for the other spec. This mocks the `request_counter` attribute in BaseRateLimiter.